### PR TITLE
Export hpo

### LINKED ISSUE #1528: Enable export on HPO term view
To facilitate the exchange of phenotypes with the clinical system, we would like to export HPO-term ids (HP:0000001) and Term names ("All") from scout. I think the current table view could actually be coerced to do that much like when exporting causatives.

### PR DESCRIPTION
**Pull request by northwestwitch**
fix #1528

**How to test**:
- Go to phenotypes page and make sure that the list can be copied and exported to Excel, same style as causatives

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by dNil
- [x] tests executed by dNil

### CODE REVIEW

**Comment by northwestwitch** [comment]
Another super easy one ready for review!

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 - Rank score model in causatives page
+- Exportable HPO terms
```
Comment (dnil) [inline CHANGELOG.md]: Well, actually they were before through the CLI. HPO terms exportable through WUI?

Comment (northwestwitch) [reply CHANGELOG.md]: Yes, of course!

**Comment by dnil** [comment]
Screenshot 2019-11-26 at 16 14 16

Not *exactly* the same thing as the CLI, but works methinks!

**Comment by northwestwitch** [comment]
> Screenshot 2019-11-26 at 16 14 16
> 
> Not _exactly_ the same thing as the CLI, but works methinks!

It was too tempting to do it the easy way! :)

**Comment by dnil** [comment]
From an edit: could we give it a filename? xslx.xslx isn't very informative.

**Comment by northwestwitch** [comment]
> From an edit: could we give it a filename? xslx.xslx isn't very informative.

Ha! I didn't notice! It should be fixed now. Thanks!

**Comment by northwestwitch** [comment]
The custom name of the export file works, I've tested it. Merging as soon as Travis completes.

**Pull request merged**